skaffold.yaml

```diff
@@ -1,63 +1,58 @@
-apiVersion: skaffold/v2beta24
+apiVersion: skaffold/v4beta2
 kind: Config
 
 requires:
   - path: examples/secrets/skaffold.yaml
+  - path: examples/values/serviceAccounts/skaffold.yaml
 
 build:
   artifacts:
     - image: substra/substra-backend
       context: .
-      docker:
-        dockerfile: docker/substra-backend/Dockerfile
       sync:
         manual:
         - src: backend/**/*
           dest: /usr/src/app
           strip: backend/
+      docker:
+        dockerfile: docker/substra-backend/Dockerfile
 
 deploy:
-  statusCheckDeadlineSeconds: 600
   helm:
     releases:
       - name: backend-org-1
         chartPath: charts/substra-backend
+        valuesFiles:
+        - examples/values/backend-org-1.yaml
         namespace: org-1
+        setValues:
+          _: _
```

Comment:
It might be good to preserve the comment from earlier (`# to support profiles`) else this is kinda mysterious.